package.json

```diff
@@ -53,7 +58,8 @@
   "main": "dist/index.js",
   "name": "@wireapp/node-addressbook",
   "os": [
-    "darwin"
+    "darwin",
+    "linux"
```

Comment:
I guess this doesnt build under linux, since there is Apple specific code for getting the addressbook

Reply:
Yes, that was by accident and removed later.

Reply:
By the way, it actually builds and runs on Linux but of course doesn't return anything so we will stay with `darwin`.